specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationTextJobsTaskStatusRequest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "displayName": "Analyze chat",
+  "createdDateTime": "2022-04-01T15:00:45Z",
+  "expirationDateTime": "2022-04-02T15:00:45Z",
+  "jobId": "3e9e8518-492f-47f9-abd1-9a7468231086",
+  "lastUpdateDateTime": "2022-04-01T15:00:49Z ",
+  "status": "succeeded",
+  "tasks": {
+    "completed": 1,
+    "failed": 0,
+    "inProgress": 0,
+    "total": 1,
+    "items": [
+      {
+        "kind": "IssueResolutionSummarizationResults",
```

Comment:
I think this just needs to be summarization results. The only mention to IRD is the values of `issue` and `resolution` in the aspects. The entire structure is generic in that regards.

Reply:
ConversationalSummarizationResults.